libbeat/cfgfile/cfgfile.go

```diff
@@ -139,24 +140,28 @@ func Load(path string, beatOverrides []ConditionalOverride) (*config.C, error) {
 
 	cfgpath := GetPathConfig()
 
-	if path == "" {
-		list := []string{}
-		for _, cfg := range configfiles.List() {
-			if !filepath.IsAbs(cfg) {
-				list = append(list, filepath.Join(cfgpath, cfg))
-			} else {
-				list = append(list, cfg)
+	if !fleetmode.Enabled() {
+		if path == "" {
+			list := []string{}
+			for _, cfg := range configfiles.List() {
+				if !filepath.IsAbs(cfg) {
+					list = append(list, filepath.Join(cfgpath, cfg))
+				} else {
+					list = append(list, cfg)
+				}
+			}
+			c, err = common.LoadFiles(list...)
+		} else {
+			if !filepath.IsAbs(path) {
+				path = filepath.Join(cfgpath, path)
 			}
+			c, err = common.LoadFile(path)
 		}
-		c, err = common.LoadFiles(list...)
-	} else {
-		if !filepath.IsAbs(path) {
-			path = filepath.Join(cfgpath, path)
+		if err != nil {
+			return nil, err
 		}
-		c, err = common.LoadFile(path)
-	}
-	if err != nil {
-		return nil, err
+	} else {
+		c = config.NewConfig()
```

Comment:
The docs for `Load` don't mention any special behavior when fleet mode is enabled. Could you update the godoc to explain the behavior under fleet mode.